src/application.js

```diff
@@ -422,6 +422,14 @@ class XtkSchemaNode {
          */
         this.nodePath = this._getNodePath(true)._path;
 
+        if (this.isRoot != undefined) {
+          this._buildLocalizationIds();
```

Comment:
Be consistent in the naming: localization id or translation id? It seems localisationId is better.